spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -38,4 +38,31 @@
       expect(response).to redirect_to('/unauthorized')
     end
   end
+
+  describe "#accept_terms_of_services" do
+    let(:user) { create(:user) }
+
+    before do
+      allow(controller).to receive_messages spree_current_user: user
+      user.spree_roles << Spree::Role.find_or_create_by(name: 'admin')
+    end
+
+    it "updates terms_of_service_accepted_at" do
+      spree_post :accept_terms_of_services, id: user.id
+
+      expect(response).to have_http_status(:ok)
+    end
+
+    context "when something goes wrong" do
+      it "returns unprocessable entity" do
```

Comment:
At first I thought it's not possible to send a 'wrong' request to this endpoint, as there's no parameters. This is testing the `if @user.update...` branch in the code, but I don't think this `update` can fail validation. If it can, it would be worth re-creating that scenario in the spec.

I see that  you've set it up to send the csrf token, so maybe that's what we're intending to cover here. 
But I think a missing/wrong csrf token is handled by Rails.

`±` In any case, the implementation ignores the response, so I think we don't need this test.